opa/datadog_checks/opa/opa.py

```diff
@@ -0,0 +1,72 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+METRIC_MAP = {
+    'http_request_duration_seconds': 'request.duration',
+}
```

Comment:
(nit) We usually define the openmetrics metrics map in a spearate `metrics.py` file

Reply:
yes, I had seen that, but as there is only one openmetrics metric that we care about, I thought of putting it here. But I will move it to `metrics.py` to be consistent